src/main/java/org/employeeBook/EmployeeController.java

```diff
@@ -0,0 +1,41 @@
+package org.employeeBook;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping("/employee")
+public class EmployeeController {
+    private final EmployeeServiceImpl employeeServiceImpl;
```

Comment:
```suggestion
    private final EmployeeService employeeServiceImpl;
```
инъекцию надо делать по интерфейсу, а не реализации